org.eclipse.Committers.json

```diff
@@ -0,0 +1,51 @@
+{
+  "id" : "org.eclipse.Committers",
+  "runtime" : "org.gnome.Sdk",
+  "runtime-version" : "3.34",
+  "sdk" : "org.gnome.Sdk",
+  "sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk11" ],
+  "command" : "eclipse",
+  "finish-args" : [ "--filesystem=host", "--share=network", "--share=ipc", "--socket=x11", "--socket=wayland", "--allow=devel", "--socket=session-bus", "--require-version=1.0.2", "--env=PATH=/app/bin:/app/jdk/bin:/usr/bin" ],
```

Comment:
Why they require the whole session bus? Mind running it with --log-session-bus and adding just the requires DBus names

Reply:
Also, one finish-arg per line makes it easier to review

Reply:
It is because the SWT webkit widget (used to serve up Javadoc and other documentation in Eclipse) fails to work without it. I have already filed a bug upstream asking them to make it use a private bus: https://bugs.eclipse.org/bugs/show_bug.cgi?id=540060

Reply:
Right, thanks!